integration-tests/src/lib.rs

```diff
@@ -84,6 +84,7 @@ fn binary_path(name: &str) -> Result<PathBuf, String> {
     let manifest_path =
         env::var_os("CARGO_MANIFEST_DIR").expect("CARGO_MANIFEST_DIR environment variable not set by cargo");
     let path = PathBuf::from(manifest_path)
+        .join("..")
```

Comment:
So this is a little dodgy. The `integration-tests` package doesn't build the binaries. The only way to really achieve this would be to copy/paste the entire contents of `src/*` into this folder, since then it'd be building the binary. This hacky way relies on the binary being built (which is why there is a build for this test run) :shrug: